pkg/pluginmanager/manager.go

```diff
@@ -167,7 +167,7 @@ func DiscoverPluginGroups(criteria *discovery.GroupDiscoveryCriteria) ([]*plugin
 		return nil, errors.New(errorNoDiscoverySourcesFound)
 	}
 
-	groups, err := discoverSpecificPluginGroups(discoveries, criteria)
+	groups, err := discoverSpecificPluginGroups(discoveries, criteria, options...)
```

Comment:
There is one other place where `discoverSpecificPluginGroups()` is called, do we know that the options are only needed when calling ` DiscoverPluginGroups()`?

Reply:
moved criteria into options